### ISSUE TITLE
Problems with Auto Scaling - not utilising new application server capacity

### ISSUE THREAD

**Issue by cnm-michael** [aws-samples/aws-scalable-big-blue-button-example#110]
After increasing the Desired Capacity in the auto scaling groups for the Application server, the new server is spun up, the IP is registered in EC2 but after creating several meetings, all meetings are started on the old server, not the new server.

This is after updating a CloudFormation stack that has been running for about 6 months, so I'm not sure if the simplest solution might be to destroy the stack and recreate it again from scratch?

**Expected behavior**
Creating several meetings through Greenlight, you can see the URL is pointing to a different application server on each meeting.

**Comment by cnm-michael** [comment]
Ok, it seems to have sorted itself out now after several hours. I just wish I knew how to tell what was going on. Because the scalelite server is on ECS, I can't SSH into it to see the status. I think I might be able to use `aws ecs exec-command` but haven't figured that out???

**Comment by couchgott** [comment]
hi @cnm-michael 

Scalelite is evaluating if an application server does still have capacity for another meeting. it might be even that you scale out your app servers scalelite still is redirecting to the first app server if resources are available. 

to check if the new app server is added to the scalelite database you can either use aws ecs exec-command on the scalealite api container or ssh into the app server and use the command from the scalelite-handler "service" as a bluepring of how to spin up an scalelite container just for the check command.  please refer to the administration section here: https://github.com/blindsidenetworks/scalelite for the check command. 

please let me know if this solves your issue with the stack :)

**Comment by cnm-michael** [comment]
hi @couchgott ,

Thanks for the suggestions. I was not able to `aws ecs exec-command` because Execute Command was not enabled on the task and I didn't want to destroy the stack to enable it.

I tried creating this other run-task command from the blueprint as you suggested:

```
aws ecs run-task --task-definition "$(aws ecs list-task-definitions --region eu-west-1 | jq -r '[ .taskDefinitionArns[] | select(contains("scalelite-handle-server")) ] | last')" \
--cluster "bbb-stack-BBBECSStack-XXXXXXX-ECSCluster" \
--region eu-west-1 --launch-type "FARGATE" \
--network-configuration "{\"awsvpcConfiguration\": {\"subnets\": [\"subnet-xxxxxa\",\"subnet-xxxxxb\",\"subnet-xxxxxc\"],\"securityGroups\": [\"sg-xxxxxd\"],\"assignPublicIp\": \"DISABLED\"}}" \
--overrides "{\"containerOverrides\": [{\"name\": \"scalelite-handle-server\",\"command\": [\"/bin/sh\", \"-c\", \"\$(./bin/rake status)\"]}]}"

```

The output comes though in CloudWatch Logs under "/bbb-stack/applicationlogs" but all I can get from the output is:

"/bin/sh: HOSTNAME: not found".

**Comment by cnm-michael** [comment]
Sorry, I've just tested terminating the servers and checking the server list in scalelite. It does get removed now, not sure why these were not being removed before. Maybe an old bug that has since been fixed. 

We had about 20+ 'offline' servers which were still enabled in scalelite but were all terminated. Seems like having these old ones lying around really slowed down adding new ones.

So basically removing all the old offline ones fixed my problem. Thanks for your help :-D
